test cases/common/182 find override/meson.build

```diff
@@ -18,7 +18,7 @@ assert(tool.found())
 assert(tool.full_path() != '')
 assert(tool.full_path() == tool.path())
 
-# six_meson_exe is an overritten project executable
+# six_meson_exe is an overridden project executable
```

Comment:
Sometimes the correct term is `overwritten`, but I _think_ it's an override and as opposed to a file on disk being written over.